GruntFile.js

```diff
@@ -94,16 +94,6 @@ module.exports = function (grunt) {
         connect: {
             uses_defaults: {}
         },
-        copy: {
```

Comment:
**thought**: I think a part of this is required for the website to have the change log.

Reply:
Yes, you're correct. I got a little trigger happy. I'll add this back in...